pliers/filters/audio.py

```diff
@@ -13,3 +15,39 @@ class AudioFilter(Filter):
 
 class AudioTrimmingFilter(TemporalTrimmingFilter, AudioFilter):
     pass
+
+
+class AudioResamplingFilter(AudioFilter):
+    
+    ''' Librosa-based audio resampling Filter.
+        Uses librosa.core.resample function.
+
+    Args:
+        target_sr (float): Target sampling rate (in Hz).
+        resample_type (str): Type of resampling. Must be one of 
+            'kaiser_best', 'kaiser_fast', 'scipy', 'fft' or 
+            'polyphase'. See librosa.core.resample documentation for 
+            more details.
+        librosa_kwargs: Optional keyword args passed onto the
+            librosa resampling function.
+    '''
+    
+    _log_attributes = ('target_sr', 'resample_type')
+    
+    def __init__(self, target_sr=22050, resample_type='kaiser_best',
```

Comment:
Is there a reason for picking `22050` as the default? Wouldn't 44 KHz be more typical? Or is the assumption that most audio probably already is at >= 44KHz, so if the user's calling this they probably want something lower?

Reply:
I was indeed assuming that 44.1 KHz would be more typical. Leaving it as it is unless you're strongly against it.

Reply:
I don't feel strongly about it, but I guess I lean towards `44100`. The reasoning is mainly that I expect this filter to be used mostly in the service of preparing audio for some other API or service that insists on a particular sampling rate. My naive guess is that the most common such rate would be 44.1 KHz. But I'm fine leaving it as is.

Reply:
default changed to 44100